packages/type-utils/src/TypeOrValueSpecifier.ts

```diff
@@ -1,3 +1,4 @@
+import { getCanonicalFileName } from '@typescript-eslint/typescript-estree';
```

Comment:
@bradzacher just confirming on architecture things - is there a better dependency setup we can use here?

Reply:
No this is probably the best way to do it and ensures we keep things in sync with the parser's logic.